javaslang/src/main/java/javaslang/collection/Vector.java

```diff
@@ -563,7 +563,7 @@ private Vector(BitMappedTrie<T> trie) {
     }
 
     @Override
-    public Vector<T> append(T element) { return appendAll(Iterator.of(element)); }
+    public Vector<T> append(T element) { return appendAll(List.of(element)); }
```

Comment:
I wish we would be able to efficiently insert just one element into the BMT without creating an new (Iterator/List) instance that is thrown away immediately. That is inefficient. We do not measure the GC effects but in a real world application GC happens and matters!